src/amp-user-notification(with_backend_service).html

```diff
@@ -0,0 +1,80 @@
+<!--
```

Comment:
This is not possible as explained here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-user-notification/amp-user-notification.md